### PR TITLE
Fixing nasty OSX bug where docker mount root dir

### PR DESCRIPTION
**Pull request by davidferlay**
Before : 
```
/bin/sh: realpath: command not found
make -s down
Removing network & containers for skilld
[+] Running 4/4
 ⠿ Container skilld_web   Removed                                                                                                                                0.4s
 ⠿ Container skilld_mail  Removed                                                                                                                                0.3s
 ⠿ Container skilld_php   Removed                                                                                                                                0.4s
 ⠿ Network skilld_front   Removed                                                                                                                                0.1s
CURDIR=/Users/skilld/Sources/ski-platform/roles/techs/services/skilld-landing/files
#
#@docker run --rm --user 0:0 -v /Users/skilld/Sources/ski-platform/roles/techs/services/skilld-landing/files:/mnt -w /mnt -e RMLIST=" web/core web/libraries web/modules/contrib web/profiles/contrib web/sites web/themes/contrib vendor" skilldlabs/php:74-fpm sh -c 'for i in $RMLIST; do rm -fr $i && echo "Removed $i"; done'
Clean-up database data from / ...
docker run --rm --user 0:0 -v /:/mnt skilldlabs/php:74-fpm sh -c "rm -fr /mnt/`basename /`"
rm: can't remove '/mnt/var/run/linuxkit-external-logging.sock': Resource busy
rm: can't remove '/mnt/var/run/linuxkit-containerd/containerd.sock': Resource busy
rm: can't remove '/mnt/var/log': Resource busy
```

After : 
```
CURDIR=/Users/skilld/Sources/ski-platform/roles/techs/services/skilld-landing/files
#@docker run --rm --user 0:0 -v /Users/skilld/Sources/ski-platform/roles/techs/services/skilld-landing/files:/mnt -w /mnt -e RMLIST=" web/core web/libraries web/modules/contrib web/profiles/contrib web/sites web/themes/contrib vendor" skilldlabs/php:74-fpm sh -c 'for i in $RMLIST; do rm -fr $i && echo "Removed $i"; done'
Clean-up database data from /Users/skilld/Sources/ski-platform/roles/techs/services/skilld-landing/files/.cache ...
docker run --rm --user 0:0 -v /Users/skilld/Sources/ski-platform/roles/techs/services/skilld-landing/files:/mnt skilldlabs/php:74-fpm sh -c "rm -fr /mnt/`basename /Users/skilld/Sources/ski-platform/roles/techs/services/skilld-landing/files/.cache`"

```

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -21,21 +21,22 @@ LOCAL_GID := $(shell id -g)
...
+CURDIR=$(shell pwd)
+
 # Define network name.
 COMPOSE_NET_NAME := $(COMPOSE_PROJECT_NAME)_front
 
 SDC_SERVICES=$(shell docker-compose config --services)
 # Determine database data directory if defined
-DB_MOUNT_DIR=$(shell cd docker && realpath $(DB_DATA_DIR))/
+DB_MOUNT_DIR=$(shell echo $(CURDIR)/$(shell basename $(DB_DATA_DIR)))
 ifeq ($(findstring mysql,$(SDC_SERVICES)),mysql)
```
Comment (andypost) [inline Makefile:33]: This curdir should be related to docker compose

Comment (davidferlay) [reply Makefile:33]: I double checked results, both commands `cd docker && realpath $(DB_DATA_DIR))/` and `$(CURDIR)/$(shell basename $(DB_DATA_DIR))` return exact same string. So this way looks ok

**Comment by andypost** [comment]
Not clear why PHP_VERSION=7 for osx added

**Comment by davidferlay** [comment]
Well that's actually a good question @andypost 

Maybe an OSX user can enlighten us ?

**Comment by andypost** [comment]
I bet it remains from past when we used to switch to php 7 from 5

**Comment by artreaktor-niks** [comment]
the main part in env extra code is
CUID=1000
CGID=1000
- before for MacOs - were need to change Make file rows
LOCAL_UID := $(shell id -u)
LOCAL_GID := $(shell id -g)
```
with
```
LOCAL_UID := 1000#$(shell id -u)
LOCAL_GID := 1000#$(shell id -g)

But for the PHP_VERSION=7 actually not sure.
This by default goes from project to project.
I think it could be removed.

**Pull request merged**